repos/rust-lang/crates-io-auth-action.toml

```diff
@@ -6,3 +6,8 @@ bots = []
 [access.teams]
 infra-admins = "write"
 infra = "triage"
+
+[[branch-protections]]
+pattern = "main"
+ci-checks = ["conclusion"]
```

Comment:
It should be capital `C` (https://github.com/rust-lang/crates-io-auth-action/blob/main/.github/workflows/ci.yml#L178).